src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -215,4 +265,24 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
 
         return getBucketItemsByType(buckets, BucketNames.COLOR, [METRIC]);
     }
+
+    private updateSuportedProperties(
+        referencePoint: IExtendedReferencePoint,
+        locationItem: any,
+    ): IExtendedReferencePoint {
+        const referencePointConfigured = cloneDeep(referencePoint);
+        if (!locationItem) {
+            return referencePointConfigured;
+        }
+        const { dfUri } = locationItem;
+        const visualizationProperties = this.visualizationProperties || {};
+        const { controls } = visualizationProperties;
```

Comment:
how about 
`const { controls = {} } = this.visualizationProperties || {};`

Reply:
updated